Networking/Networking/Model/Product/StoredProductSettings.swift

```diff
@@ -0,0 +1,38 @@
+import Foundation
+
+/// Models a pair of `siteID` and Product Settings
+/// These entities will be serialised to a plist file
+///
+public struct StoredProductSettings: Codable, Equatable {
```

Comment:
Good question @shiki. 
In reality, there is a reason why I put it under Networking. Networking and Storage targets don't know each other, because there is the Yosemite layer that puts them in communication. 
If I move the `StoredProductSettings` model, Storage should import Networking because of the other models used inside it like `ProductStockStatus`, `ProductStatus` and `ProductType`. Did you solve the compilation issues importing the Networking framework?

About `GeneralAppSettings`, it's correct to keep in Storage because they don't need to import models declared in Networking. :nod:

Reply:
Doh! You're right. I had to import `Networking` inside `Storage`. 🤦 

I guess it's not really that important right now. I think we need to consolidate where these models are located at some point.